iterative/resource_machine.go

```diff
@@ -122,6 +122,14 @@ func machineSchema() *map[string]*schema.Schema {
 			Optional: true,
 			Default:  "",
 		},
+		"metadata": &schema.Schema{
+			Type:     schema.TypeMap,
+			ForceNew: true,
+			Optional: true,
+			Elem: &schema.Schema{
+				Type: schema.TypeString,
+			},
+		},
```

Comment:
```suggestion
		"metadata": &schema.Schema{
			Type:     schema.TypeMap,
			ForceNew: true,
			Optional: true,
			Elem: &schema.Schema{
				Type: schema.TypeString,
			},
		},
```

Removing in favor of https://github.com/iterative/terraform-provider-iterative/pull/193